docs/stats.md

```diff
@@ -0,0 +1,30 @@
+## accessing stats
+
+stats can be accessed via 
+1. `pumactl`
```

Comment:
I think these are the ways you can access stats:

```
1. `pumactl stats`, assuming puma was started with the use of the control/status server
2. `GET /stats` request to the control/status server directly
3. `Puma.stats` from within the Ruby process
```

Reply:
yeah I'd prefer to see `$ pumactl stats` as the way we refer to that, just to make it clear its a CLI command